frontend/src/assets/i18n/fi.json

```diff
@@ -149,7 +152,10 @@
     "CREATE": "Luo joukkue",
     "EDIT": "Muokkaa joukkuetta",
     "NAME": "Nimi",
-    "TEAM_N_DELETED": "Joukkue {{name}} poistettu"
+    "TEAM_N_DELETED": "Joukkue {{name}} poistettu",
```

Comment:
**thought:** no need to change this now but in general would be good to not replicate the context key in the actual keys meaning these look now like `TEAMS.TEAM_MEMBERS` with double "TEAM"